servicetalk-http-api/src/main/java/io/servicetalk/http/api/MultiAddressHttpClientFilterFactory.java

```diff
@@ -27,9 +28,17 @@
 /**
  * A factory for {@link StreamingHttpClientFilter} to filter clients for different unresolved addresses.
  *
+ * @deprecated Use
```

Comment:
should we deprecate in 0.41.x and remove in this PR (or followup PR for 0.42)? looks like this PR removes all practical usages of this interface so it won't be very useful anymore (and other APIs related to it have already been broken).

Reply:
Correct, this class is now removed.